Example/Example for Delirium/Modules/StickyCollectionViewLayout/StickyCell.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  StickyCell.swift
+//  Delirium
+//
+//  Created by Stijn Willems on 30/11/2016.
+//  Copyright © 2016 CocoaPods. All rights reserved.
+//
+
+import UIKit
+
+class StickyCell: UICollectionViewCell {
+    @IBOutlet weak var label: UILabel!
+
+    override func layoutSubviews() {
+        layer.borderColor = UIColor.black.cgColor
+        layer.borderWidth = 1
+        layer.cornerRadius = 4
+
+        super.layoutSubviews()
```

Comment:
❓Is there a reason why you set the border's properties before the `super.layoutSubviews()` call?